src/chunk.c

```diff
@@ -135,6 +135,7 @@ static Chunk *get_chunks_in_time_range(Hypertable *ht, int64 older_than, int64 n
 #define ASSERT_IS_NULL_OR_VALID_CHUNK(chunk) Assert(chunk == NULL || IS_VALID_CHUNK(chunk))
 
 #define CHUNK_STATUS_DEFAULT 0
+#define CHUNK_STATUS_COMPRESSED 1
```

Comment:
Could you add a comment here that describes how this gets used for compressed chunks: specifically compressed_chunk_id may not be set but status flag would be still set for distributed chunks